docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -44,7 +44,31 @@ $ docker build -t alluxio .
 By default, this will build an image for the latest released version of Alluxio. To build
 from a local Alluxio tarball instead, you can use `--build-arg`
 ```bash
-docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+$ docker build -t alluxio --build-arg ALLUXIO_TARBALL=alluxio-snapshot.tar.gz .
+```
+
+### Set up ramdisk to enable short-circuit reads
+
+When the Alluxio client runs local to an Alluxio worker, a shared ramdisk should be set
```

Comment:
Does this mean an alluxio client is in the same container as the worker, or a different container on the same host? What does "local" mean in this context?